flowzone.yml

```diff
@@ -157,6 +157,16 @@ on:
         type: string
         required: false
         default: ""
+      github_make_final:
+        description: "Finalise releases on merge."
+        type: boolean
+        required: false
+        default: true
+      github_make_latest:
+        description: "Move latest pointer on merge."
+        type: boolean
+        required: false
+        default: true
       restrict_custom_actions:
```

Comment:
These options seem overly specific to the github job implementation details. Maybe there is a way to merge them into one, e.g. `github_prerelease`, where `make_latest = !github_prerelease` or something like that?